server/src/controllers/api/makeMove.js

```diff
@@ -47,6 +51,39 @@ module.exports = async (req, res) => {
     return error.sendResponse(res);
   }
 
+  if (fieldsToUpdate.result) {
+    let whitePlayerProfit, blackPlayerProfit;
+
+    if (fieldsToUpdate.result === 'white') {
```

Comment:
Switch case
Sides should be const
Amount of money shouldn't be represented by numbers (https://jeffreyeverhart.com/2020/09/28/javascript-tips-using-named-constants-for-magic-numbers/#:~:text=What%20Are%20Magic%20Numbers%3F,replaced%20by%20a%20named%20constant.&text=Date.,-.)